lib/smart_answer_flows/benefit-cap-calculator/questions/receiving_non_exemption_benefits.govspeak.erb

```diff
@@ -2,24 +2,7 @@
   Do you or someone in your household get any of the following benefits:
 <% end %>
 
-<% options(
-  "bereavement": "Bereavement Allowance",
-  "carers": "Carer's Allowance",
-  "child_benefit": "Child Benefit",
-  "child_tax": "Child Tax Credit",
-  "esa": "Employment and Support Allowance",
-  "guardian": "Guardian's Allowance",
-  "incapacity": "Incapacity Benefit",
-  "income_support": "Income Support",
-  "jsa": "Jobseeker’s Allowance",
-  "maternity": "Maternity Allowance",
-  "sda": "Severe Disablement Allowance",
-  "widowed_mother": "Widowed Mother's Allowance",
-  "widowed_parent": "Widowed Parent's Allowance",
-  "widow_pension": "Widow's Pension",
-  "widows_aged": "Widow's Pension (age related)",
-  "none_above": "None of the above"
-) %>
```

Comment:
I'd prefer to see this merged with commit 3ac69aaed833650a6b120de85d013fbd2a84b614 ("Refactor Benefit cap calculator (Making benefit rates and type configurable)") as mentioned in a comment on that commit.